src/Chapter1.hs

```diff
@@ -614,8 +621,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = 
+  let absValue = abs n
+      units = mod absValue 10
+      dozen = mod (div (absValue - units) 10) 10
```

Comment:
Nice one! 👌🏼 

Just an additional note (no action required, this is just FYI 🙂 ).
You could reuse the `lastDigit` function from the previous task in these two. You can notice that both `units` and `dozen` have `mod ... 10`.

Reply:
WOW, off course I change